flink-python/src/main/resources/META-INF/NOTICE

```diff
@@ -67,11 +68,20 @@ The bundled Apache Beam dependencies bundle the following dependencies under the
 - io.netty:netty-tcnative-boringssl-static:2.0.26.Final
 - io.opencensus:opencensus-api:0.24.0
 - io.opencensus:opencensus-contrib-grpc-metrics:0.24.0
-- net.bytebuddy:1.10.8
+- io.perfmark:perfmark-api:0.19.0
+- net.jpountz.lz4:lz4:1.3.0
 
 The bundled Apache Beam dependencies bundle the following dependencies under the BSD license.
 See bundled license files for details
 
 - com.google.auth:google-auth-library-credentials:0.18.0
+- com.google.protobuf.nano:protobuf-javanano:3.0.0-alpha-5
 - com.google.protobuf:protobuf-java:3.11.0
 - com.google.protobuf:protobuf-java-util:3.11.0
+- com.jcraft:jzlib:1.1.3
+
+The bundled Apache Beam dependencies bundle the following dependencies under the Bouncy Castle license.
+See bundled license files for details
+
+- org.bouncycastle:bcpkix-jdk15on:1.54
```

Comment:
It would be good to document somewhere (maybe the licensing wiki page) how these were determined.

I assume you pulled them from https://github.com/apache/beam/blob/release-2.23.0/buildSrc/src/main/groovy/org/apache/beam/gradle/GrpcVendoring_1_26_0.groovy ?
If so, what about these:
```
      "org.eclipse.jetty.alpn:alpn-api:$alpn_api_version",
      "org.eclipse.jetty.npn:npn-api:$npn_api_version",
      "org.jboss.marshalling:jboss-marshalling:$jboss_marshalling_version",
      "org.jboss.modules:jboss-modules:$jboss_modules_version"
```

Reply:
Yes, you are right. jetty and jboss are not needed and so excluded from the fat jar.